x-pack/test/cases_api_integration/security_and_spaces/tests/common/cases/migrations.ts

```diff
@@ -476,5 +477,38 @@ export default function createGetTests({ getService }: FtrProviderContext) {
         });
       });
     });
+
+    describe('8.7.0', () => {
+      before(async () => {
+        await kibanaServer.importExport.load(
+          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.5.0/cases_severity.json'
+        );
+      });
+
+      after(async () => {
+        await kibanaServer.importExport.unload(
+          'x-pack/test/functional/fixtures/kbn_archiver/cases/8.5.0/cases_severity.json'
+        );
+        await deleteAllCaseItems(es);
+      });
+
+      describe('severity', () => {
+        it('severity string labels are converted to matching number', async () => {
+          const expectedSeverityValues: Record<string, ESCaseSeverity> = {
+            'cases:063d5820-1284-11ed-81af-63a2bdfb2bf6': ESCaseSeverity.LOW,
+            'cases:063d5820-1284-11ed-81af-63a2bdfb2bf7': ESCaseSeverity.MEDIUM,
+            'cases:063d5820-1284-11ed-81af-63a2bdfb2bf8': ESCaseSeverity.HIGH,
+            'cases:063d5820-1284-11ed-81af-63a2bdfb2bf9': ESCaseSeverity.CRITICAL,
+          };
+
+          const casesFromES = await getCaseSavedObjectsFromES({ es });
+
+          for (const hit of casesFromES.body.hits.hits) {
+            const caseID = hit._id;
+            expect(hit._source?.cases.severity).to.eql(expectedSeverityValues[caseID]);
```

Comment:
nit: let's add this check first `expect(expectedSeverityValues[caseID]).not.to.be(undefined)`